### PR TITLE
Anton/contribute delete migrated

### PR DESCRIPTION
**Pull request by antondelchev**
Added method to delete a migrated data set. Related to #67

### CODE REVIEW

File: src/zos_files/zowe/zos_files_for_zowe_sdk/files.py [hunk 1]
```diff
@@ -485,3 +486,43 @@ def list_unix_file_systems(self, file_path_name=None, file_system_name=None):
...
+            "wait": json.dumps(False),
+            "purge": json.dumps(False),
+        }
+
+        if wait:
+            data["wait"] = json.dumps(True)
+        
+        if purge:
+            data["purge"]: json.dumps(True)
+
```
Comment (skorzhevsky) [inline src/zos_files/zowe/zos_files_for_zowe_sdk/files.py:522]: Maybe just:
    data = {
            "request": "hdelete",
            "wait": json.dumps(wait),
            "purge": json.dumps(purge),
        }

Comment (antondelchev) [reply src/zos_files/zowe/zos_files_for_zowe_sdk/files.py]: Thank you!

**Review by t1m0thyj — APPROVED** [review]

**Review by boris-bc — APPROVED** [review]

File: src/zos_files/zowe/zos_files_for_zowe_sdk/files.py [hunk 2]
```diff
@@ -567,6 +567,39 @@ def list_unix_file_systems(self, file_path_name=None, file_system_name=None):
         response_json = self.request_handler.perform_request("GET", custom_args, expected_code=[200])
         return response_json
 
+    def delete_migrated_data_set(self, dataset_name: str, wait=False, purge=False):
```
Comment (boris-bc) [inline src/zos_files/zowe/zos_files_for_zowe_sdk/files.py:570]: I think the order of those arguments should be swapped, first `purge` and then `wait` as last. If you do this change, review your parametrized test that it is still structured correctly.

**Review by boris-bc — CHANGES_REQUESTED** [review]

File: src/zos_files/zowe/zos_files_for_zowe_sdk/files.py (hunk 2 above)
Comment (antondelchev) [reply src/zos_files/zowe/zos_files_for_zowe_sdk/files.py:570]: Ok, will swap them and refactor the tests.

**Review by boris-bc — APPROVED** [review]

**Pull request merged**